specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-07-01/netapp.json

```diff
@@ -0,0 +1,2355 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft NetApp",
+    "description": "Microsoft NetApp Azure Resource Provider specification",
+    "version": "2019-07-01",
+    "x-ms-code-generation-settings": {
+      "name": "AzureNetAppFilesManagementClient"
+    }
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.NetApp/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "summary": "Describes the Resource Provider",
+        "description": "Lists all of the available Microsoft.NetApp Rest API operations",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/checkNameAvailability": {
+      "post": {
+        "summary": "Check resource name availability",
+        "description": "Check if a resource name is available.",
+        "operationId": "CheckNameAvailability",
```

Comment:
Needs to follow the Noun_Verb format else you get language-specific class names and loose consistency (the noun is used to generate the class name).  Maybe `NetApps_CheckNameAvailability`?  Open to other ideas.

Reply:
ok. Note that this does not relate to the change made and exists in earlier versions. Can it therefore be delayed until the next version since this swagger is already now overdue?

Reply:
I see.  If there are previous releases with this pattern then let's just leave it as-is in order to not introduce any breaking changes.